packages/tc-ui/src/primitives/relationship/lib/rich-relationship-properties.jsx

```diff
@@ -5,46 +5,39 @@ const {
 } = require('../../../lib/mappers/component-assigner');
 const { getValue } = require('../../../lib/mappers/get-value.js');
 
-class RelationshipProperties extends React.Component {
-	constructor(props) {
-		super(props);
-		this.props = props;
-	}
+const RelationshipProperties = props => {
+	const { value, type, properties, onChange } = props;
+	const propertyfields = Object.entries(properties);
 
-	render() {
-		const { value, type, properties } = this.props;
-		const propertyfields = Object.entries(properties);
+	return propertyfields
+		.filter(
+			([, schema]) =>
+				// HACK: need to get rid of fields that are doing this
+				!schema.label.includes('deprecated') &&
```

Comment:
you can get rid of lines 15 & 16 - I have tidied up the deprecated stuff in the schema